v2/internal/frontend/desktop/linux/window.go

```diff
@@ -773,6 +779,17 @@ func (w *Window) IsMaximised() bool {
 	return result > 0
 }
 
+func (w *Window) IsMinimized() bool {
+	result := C.IsMinimized(w.asGTKWidget())
+	return result > 0
+}
+
+func (w *Window) IsNormal() bool {
+	max := C.IsMaximised(w.asGTKWidget())
+	min := C.IsMinimized(w.asGTKWidget())
+	return !(max > 0) && !(min > 0)
```

Comment:
Could this be simplified to `max == 0 && min == 0` ?

Reply:
is changed